loadgen/cmd/otelbench/README.md

```diff
@@ -0,0 +1,129 @@
+# otelbench
+
+otelbench wraps the collector inside a Go test benchmark. It outputs throughput numbers in standard Go benchmark format. It uses a collector config yaml as a base, and apply some overrides from command line options.
+
+## Usage
+
+```
+Usage of ./otelbench:
+  -api-key string
+        API key for target server
+  -config string
+        path collector config yaml (default "config.yaml")
+  -endpoint value
+        target server endpoint for both otlp and otlphttp exporters (default to value in config yaml), equivalent to setting both -endpoint-otlp and -endpoint-otlphttp
+  -endpoint-otlp value
+        target server endpoint for otlp exporter (default to value in config yaml)
+  -endpoint-otlphttp value
+        target server endpoint for otlphttp exporter (default to value in config yaml)
+  -exporter-otlp
+        benchmark exporter otlp (default true)
+  -exporter-otlphttp
+        benchmark exporter otlphttp (default true)
+  -header value
+        extra headers to use when sending data to the server
```

Comment:
Maybe describe the expected format here? Is it k=v, and you can repeat the flag?